unittests/forked_tests.cpp

```diff
@@ -3,22 +3,27 @@
 
 #include <eosio/chain/fork_database.hpp>
 
-#include <fc/variant_object.hpp>
-
 #include <boost/test/unit_test.hpp>
 
-#include <contracts.hpp>
-#include <test_contracts.hpp>
-
 #include "fork_test_utilities.hpp"
 
 using namespace eosio::chain;
 using namespace eosio::testing;
 
-// Tests in this file are for Legacy only. Savanna forked tests will be done 
-// by https://github.com/AntelopeIO/spring/issues/196
+// ---------------------------------------------------
+// Following tests in this file are for Legacy only:
+//    - fork_with_bad_block
+//    - forking
+//    - prune_remove_branch
+//    - irreversible_mode
+//    - push_block_returns_forked_transactions
+//
+// Similar Savanna tests are in: `forked_tests_if.cpp`
+// ---------------------------------------------------
+
 BOOST_AUTO_TEST_SUITE(forked_tests)
 
+// ---------------------------- irrblock ---------------------------------
 BOOST_AUTO_TEST_CASE( irrblock ) try {
    legacy_tester c;
```

Comment:
This is also a test that appears to only use `legacy_tester` but is not in the list in your comment above. But, this `irrblock` test case (which is also badly named) is so pointless that I am fine with just deleting it.

Reply:
Sounds good, removing it.